templates/endpoints_v2/README.md

```diff
@@ -14,17 +14,25 @@ You can also find more advanced tutorials in the `examples/` folder, including t
 
 ## Step 1 - Run the model locally in the Workspace
 
-The llm-serve.yaml file in this example runs the Mistral-7B model. There are 2 important configurations you would need to modify:
-1. The `models` config in `llm-serve-*.yaml` contains a list of YAML files for the models you want to deploy. You can run any of the models in the `models` directory or define your own model YAML file and run that instead. All config files follow the naming convention `{model_name}_{accelerator_type}_{tensor_parallelism}`. Follow the CustomModels [guide](examples/CustomModels.ipynb) for bringing your own models.
-2. `HUGGING_FACE_HUB_TOKEN` - The Meta Llama-2 family of models need the HUGGING_FACE_HUB_TOKEN variable to be set to a Hugging Face Access Token for an account with permissions to download the model.
+We provide a starter command to allow you to run Llama-2 and Mistral-family models via Ray Serve. You can specify the model ID, GPU type, tensor parallelism via the command arguments. You can also follow the [guide](examples/CustomModels.ipynb) for bringing your own models.
```

Comment:
Can we also keep some yamls as example configs?

Reply:
Some config yamls are still kept. I didn't remove everything